### PR TITLE
Spearfishing Rebalance: Unnerfing Bronze Age Weaponry

### PR DESCRIPTION
**Pull request by Snakebittenn**
Spears have been brought back to nearly their previous power. Unsure how 0.75 would go, so went with 0.50. Half-revert of #6433

### CODE REVIEW

**Comment by Snakebittenn** [comment]
!review

File: code/game/objects/items/weapons/material/twohanded.dm [hunk 1]
```diff
@@ -227,9 +227,9 @@
 	force = 10
 	w_class = 4.0
 	slot_flags = SLOT_BACK
-	force_divisor = 0.35 // 21 damage for steel (hardness 60)
-	unwielded_force_divisor = 0.2 // 12 damage for steel (hardness 60)
-	thrown_force_divisor = 1.2 // 24 damage for steel (weight 20)
+	force_divisor = 0.50 // 30 damage for steel (hardness 60)
+	unwielded_force_divisor = 0.15 // 15 damage for steel (hardness 60)
```
Comment (alsoandanswer) [inline code/game/objects/items/weapons/material/twohanded.dm:231]: isn't...0.15 smaller than 0.2

**Comment by Kaedwuff** [comment]
No.

**Comment by Arrow768** [comment]
Create a feedback topic an explain the reasoning behind the revert.

**Comment by Snakebittenn** [comment]
https://forums.aurorastation.org/topic/14616-spear-nerf-partial-revert/

**Comment by NonQueueingMatt** [comment]
Closing because of the absence of good feedback, and in general this isn't quite something we want.

**Pull request closed without merging**